Salesforce/Module2/Soql_Queries.txt

```diff
@@ -0,0 +1,58 @@
+					SOQL:
+
+Query 1:
+	SELECT FirstName__c, LastName__c, DOB__c, Sex__c 
+	FROM Student__c 
+	WHERE Name LIKE 'S%''
+
+Query 2:
+	SELECT FirstName__c,Name, Class__r.Name FROM Student__c 
+	WHERE Class__r.Name LIKE '10%'
+
+Query 3:
+	SELECT FirstName__c,LastName__c, Class__r.Name FROM Student__c 
+
+Query 4:
+	SELECT Name , (SELECT FirstName__c, LastName__c FROM Students__r) FROM Class__c
+
+Query 5: 
+	SELECT FirstName__c, Class__r.ClassTeacher__r.Name 
+	FROM Student__c
+	WHERE CLass__r.ClassTeacher__r.Name LIKE 'S%'
+
+Query 6:
+	SELECT FirstName__c,LastName__c
+	FROM Student__c
+	WHERE CALENDAR_YEAR(DOB__c) > 2000
+
+Query 7: 
+	 SELECT LastModifiedDate,Name FROM Lead Where LastModifiedDate > THIS_MONTH
+	 
+Query 8: 
+	SELECT Name, (SELECT FirstName__c FROM Students__r WHERE FirstName__c LIKE 'S%') FROM Class__c WHERE id IN 
+	(SELECT Class__c FROM Student__c WHERE FirstName__c LIKE 'S%')
+	
+Query 9: 
+	SELECT Name FROM Contact WHERE id IN (SELECT Teacher__c from Teach__c WHERE subject__c = 'Hindi') AND id IN (SELECT Teacher__c from Teach__c WHERE subject__c = 'English') 
+	
+Query 10:
+	SELECT FirstName__c, Name FROM Student__c
+	WHERE Class__c IN (SELECT class__c from Teach__c WHERE Subject__c='Hindi' OR Subject__c='Science')
```

Comment:
As per the question, query 'All students who have hindi or maths'. 
The above query needs to be updated.